modules/swagger-models/src/main/java/io/swagger/models/Response.java

```diff
@@ -67,8 +67,9 @@ public void setSchema(Property schema) {
         return this.examples;
     }
 
-    public void setExamples(Map<String, Object> examples) {
+    public Response setExamples(Map<String, Object> examples) {
         this.examples = examples;
+        return this;
```

Comment:
we don't want to modify setter behaviour, see also comment above

Reply:
Sure.